### DISCUSSION TITLE
How to disable loading of images?

### DISCUSSION THREAD

**Discussion by tobwen** [daijro/camoufox#13] [Q&A] [answered]
Is there a way to disable loading of images? I know the adblocker supports this... do I need to to configure it somehow or is there a switch to make Camoufox not load images/media at all?

Edit:
Maybe this one? https://www.zenrows.com/blog/blocking-resources-in-playwright

**Comment by daijro** [comment, marked as answer]
Hello!

The most effective way to block images is through the `permissions.default.image` Firefox preference:
https://kb.mozillazine.org/Permissions.default.image

```python
with Camoufox(
    headless=False,
    firefox_user_prefs={
        "permissions.default.image": 2,
    }
) as browser:
    page = browser.new_page()
    page.goto("https://www.pexels.com/")
```

---

Edit: As of v0.1.3, you can now also pass `block_images=True`:

```python
with Camoufox(
    headless=False,
    block_images=True
) as browser:
    page = browser.new_page()
    page.goto("https://www.pexels.com/")
```

**Comment by moonlabsai** [comment]
What about videos?

**Reply by ioc2e3** [reply]
> What about videos?

Same problem ...

.

**Reply by KittisakPh** [reply]
@ioc2e3 hi but I see you ingesting on CloakBrowser is better camoufox for real can u share experience????

**Reply by ioc2e3** [reply]
> @ioc2e3 hi but I see you ingesting on CloakBrowser is better camoufox for real can u share experience????

Sorry ...

In addition to watching videos on platforms such as YouTube ...

I rarely use computer ...

So , I don't know how to share experience ...

.

**Reply by KittisakPh** [reply]
@ioc2e3 No, I want to know if the difference you experienced between using Camoufox and CloakBrowser was significant, and whether the claim that CloakBrowser is better than Camoufox is true.

**Reply by KittisakPh** [reply]
@ioc2e3 A better way to avoid bots, as he claims is really

**Reply by ioc2e3** [reply]
> @ioc2e3 No, I want to know if the difference you experienced between using Camoufox and CloakBrowser was significant, and whether the claim that CloakBrowser is better than Camoufox is true.

That ...

I'm not sure about that ...

.
